src/swap/components/SwapMessage.tsx

```diff
@@ -25,7 +26,10 @@ export function SwapMessage() {
 
   return (
     <div className="flex pt-2">
-      <span className={text.label2} data-testid="ockSwapMessage_Message">
+      <span
+        className={cn(text.label2, className)}
+        data-testid="ockSwapMessage_Message"
+      >
```

Comment:
Left as a Span element. Can update to a div if preferred!

Reply:
nit: let's combine the `div` and `span` into a single element

Reply:
Updated